.github/workflows/build-test.yaml

```diff
@@ -205,35 +205,39 @@ jobs:
         run: tests/run-master-in-k3s.sh local/freeipa-server:${{ matrix.os }} freeipa-server-${{ matrix.os }}.tar
 
   push-after-success:
-    name: Push images to Docker Hub
+    name: Push images to Registry
     runs-on: ubuntu-20.04
     needs: [ test-docker, test-podman, test-rootless-podman, test-upgrade, test-k3s ]
-    if: github.event_name != 'pull_request' && github.repository == 'freeipa/freeipa-container' && github.ref == 'refs/heads/master'
+    if: github.event_name != 'pull_request' && ( ( github.repository == 'freeipa/freeipa-container' && github.ref == 'refs/heads/master' ) || github.repository != 'freeipa/freeipa-container' )
```

Comment:
This `if` does not seem correct -- it does not take into account if `DOCKER_IMAGE_BASE` is set or not ... which it should or you will get failures down the road.

Reply:
But that `exit 2` will cause the job and therefore the whole workflow fail, won't it?
You likely want some separate job which will set `outputs` accordingly and then in `push-after-success`'s `if` check the `outputs` of the previous job. That way you will be able to skip without causing failures.